include/distconfig.php

```diff
@@ -410,6 +410,18 @@
 
 // }}}
 
+// {{{ Mardown Render
+
+// Uncomment this line if you want to enable Mardown Rendering of README.md file in the path. 
+// You will need the Parsedown.php library for this to work. You can either manually add this 
```

Comment:
Suggested improvement:

> You will need the Parsedown.php (https://github.com/erusev/parsedown) library for this to work.

People should now where to get the file from without Composer.

Reply:
Done. Agreed.